backend/src/apiserver/resource/model_converter.go

```diff
@@ -253,34 +255,19 @@ func runtimeConfigToModelParameters(runtimeConfig *api.PipelineSpec_RuntimeConfi
 	if runtimeConfig == nil {
 		return "", nil
 	}
-	var params util.SpecParameters
+	// Use structpb to marshal protobuff value, store them in a map, then marshal the entire map into a single new string
```

Comment:
I wonder if this is this necessary. Can you just json marshal the runtimeConfig.Parameters of type  map[string]*structpb.Value without marshaling the value first?

Reply:
That's right. I updated the code accordingly.